src/ui/wxWidgets/SafeCombinationEntryDlg.cpp

```diff
@@ -373,8 +373,14 @@ void SafeCombinationEntryDlg::ProcessPhrase()
     const stringT fname(m_filename.c_str());
     stringT locker(L"");
     if (!m_readOnly && !m_core.LockFile(fname, locker)) {
-      errmess = _("Could not lock file, opening read-only\nLocked by ");
+      errmess = _("Could not lock file, opening read-only.\n");
+
+      if (PWSUtil::HasValidLockerData(locker)) {
+        errmess += _("Locked by ");
```

Comment:
Minor: line 382 could be combined with this one to append both strings in one go.

Reply:
The reason for this split is that 'locker' can provide different information.

Usually, if a lock file has been detected the resulting error message in the message box would appear as follows.

> Could not lock file, opening read-only.
> Locked by \<user@machine:nnnnnnnn>

If an error occured during the processing of the lock file 'locker' would refer to an corresponding error message.
This would lead to the following result.

> Could not lock file, opening read-only.
> Locked by \<error message>

With the current implementation it should be something as the following.

> Could not lock file, opening read-only.
> \<specific error message or reason for the read only mode>

I would have liked to leave the original string (line 376) untouched so as not to invalidate the translations.

Reply:
Makes sense, I didn't consider the translations.